src/themes/mainTheme.ts

```diff
@@ -83,11 +83,6 @@ export default createMuiTheme({
     MuiExpansionPanelDetails: {
       root: { background: Colors.ExpandedPanel },
     },
-    MuiFormControl: {
-      root: {
-        width: '100%',
-      },
-    },
     MuiFormLabel: {
       root: {
```

Comment:
off topic; men det er kanskje her en plass vi må sette farge på teksten for at label til RadioGroup ikke skal bli lilla på focus?